postreise/plot/plot_states.py

```diff
@@ -65,109 +65,156 @@ def get_state_borders():
 
 
 def plot_states(
-    state_list,
-    color_list,
-    labels_list,
-    leg_color,
-    leg_label,
-    font_size,
+    state_list=None,
+    colors=None,
+    line_color="grey",
+    line_width=1,
+    fill_alpha=0.8,
+    labels_list=None,
+    legend_colors=None,
+    legend_labels=None,
+    legend_title=None,
+    font_size="10pt",
+    bokeh_figure=None,
     web=True,
     us_states_dat=None,
+    background_map=True,
 ):
     """Plots US state borders and allows color coding by state,
         for example to represent different emissions goals.
 
     :param list state_list: list of us states to color code.
-    :param list color_list: list of colors associated with states in state_list.
-    :param list labels_list: list of labels for us states.
-    :param list leg_color: list of colors for legend
-    :param list leg_label: list of labels for colors in legend
+    :param str/iterable colors: color or colors associated with states in state_list.
+    :param str line_color: color of state outlines.
+    :param int/float line_width: thickness of state outlines.
+    :param int/float fill_alpha: fill alpha for state patches.
```

Comment:
Should we say transparency for state patches instead

Reply:
Your english is good

Reply:
Done